### ISSUE TITLE
Error creating query engine Drools in Java Dynamic Web Project

### ISSUE THREAD

**Issue by ayeshaafzal** [protegeproject/swrlapi#53]
I am working with SWRLAPI and trying to invoke SWRLRuleEngine in a Java Faceted Project (Server Runtime: Apache Tomcat) using: 

`SWRLRuleEngine ruleEngine = SWRLAPIFactory.createSWRLRuleEngine(ontology);`

Getting the following exception:

` org.swrlapi.exceptions.SWRLRuleEngineException: Error creating query engine Drools. Exception: org.swrlapi.exceptions.TargetSWRLRuleEngineInternalException. Message: internal error generating Drools rule
rule prp_irp when IROPA($p:pid) OPAA($s:s, pid==$p, $x:o) then inferrer.inferFalse("PRP_IRP", $x.id, $p); end
[Rule Compilation error : [Rule name='prp_irp']
	defaultpkg/Rule_prp_irp1903926907.java (7:3678) : The method inferFalse(String, String[]) in the type DroolsOWLAxiomHandler is not applicable for the arguments (String, String, String)
]`

My objective is to run SWRL Rules through JSP scriptlet or Servlet in order to create a web based interface for the project. The SWRLAPI example code works if I don't add Apache Tomcat to the server runtime. 
Looking for a quick workaround to resolve this conflict between SWRL Rule engine and Tomcat.

**Comment by martinjoconnor** [comment]
Are you using the latest version of the SWRLAPI?

Can you post the ontology that produces this error. (Change extension from .owl to .txt because GitHub does not recognize OWL files.)

**Comment by ayeshaafzal** [comment]
Thank you Martin for your quick response.

I think I am using the latest SWRLAPI. I just followed the instructions for the swrlapi-example that I downloaded from https://github.com/protegeproject/swrlapi-example.

Using the same project however I have updated the project properties to make it a faceted project to convert it to "Dynamic Web Project". In addition I have added Apache Tomcat to server runtime.

Is there a way to share the ontology file privately?
